docs/0.2.1-upgrade-to-0.3.3.md

```diff
@@ -0,0 +1,48 @@
+# Upgrading 0.2.1 version
+
+## Prerequisites
+
+* A networked Admin tails workstation configured with the SecureDrop repo cloned to the `Persistent` directory.
+
+* Physical/remote access to the app and monitor servers
+
+## Upgrade 0.2.1 steps
+
+There is a `0.2.1_collect.py` python script that will backup a SecureDrop version 0.2.1 app server. This script will need to be ran on the app server prior to performing a clean install of SecureDrop 0.3.x. The direction below assume that the admin has physical access to the app server.
+
+### On the App server
+
+* Install git
+
+`sudo apt-get install git`
+
+* Clone the SecureDrop repo
+
+`git clone https://github.com/freedomofpress/securedrop`
+
+* Change directory into the securedrop folder
+
+`cd securedrop`
+
+* Checkout and verify the current release
+
+`git checkout tags/0.3.3`
```

Comment:
Hm, are you sure you based this on the latest version of the [Hackpad](https://securedrop.hackpad.com/Upgrade-to-0.3.3-jis3fmQ3AnE)? I made some edits that I am not seeing in this document. For example, `tags/` is unnecessary, so I changed this line to `git checkout 0.3.3` in the Hackpad.